src/base/_defaults.scss

```diff
@@ -151,3 +151,26 @@ table {
   text-align: left;
   width: 100%;
 }
+
+/*
+ * 1. Removes default margin
+ * 2. Center content
+ */
+
+figure {
+  margin: 0; /* 1 */
+
+  & > * {
+    margin-left: auto; /* 2 */
+    margin-right: auto; /* 2 */
+  }
+}
+
+/* Apply default styles to <figcaption> */
+
+figcaption {
+  font-size: ms.step(-1);
+  font-style: italic;
+  margin-top: ms.step(-1);
```

Comment:
We may want to wrap this in something in case `figcaption` is not the last child. (Believe it or not, it's actually allowed to have `figcaption` be first or last in a `figure` element!)

Reply:
Do you think all the caption styles should be wrapped or just the `margin-top` that provides space between the `figure` element and `figcaption`?

Reply:
Just the `margin-top`. You could do something like this if you wanted to:

```scss
figcaption {
  // etc.

  &:last-child {
    margin-top: ms.step(-1);
  }
}
```